template/defaults.yaml

```diff
@@ -2,7 +2,7 @@
 # vim: ft=yaml
 ---
 template:
-  pkg: template
+  pkg: bash
```

Comment:
I think that the way it was before allows us to kept two clearly differentiated groups of files:

* the *formula-related* ones, which are all of those that have `template-something` or `something-template` or 'template' in the name.
* the *testing-related* ones, which are not intrinsically needed by the formulas: that's where (I think) we should put these examples, like `pkg: bash` or `service: whatever` 

I think the former group should be respected and kept free of 'testing-related' content, while the latter are the place for whatever we consider useful for testing (in fact, this is the same reason why I think that the linting and semver tools/files should be moved under `test`, at least).

If the content of the **formula-related** files is named `template` then I can simply go with a 'search & replace' for 's/template/my_package/g', the usage of a 'templated formula' is clear, and the entry-level for newcomers is easy (I usually do this, in fact :smile:).

However, if a newcomer needs to use the formula, 'search manually all around and make sense what to replace and what not, what is a requirement and what is an example', the template loses part of its value: 

'what should I replace? `pkg: bash` in the `template/defaults.yaml`? or that is a dependency that should not be modified?' 'same goes to `service: systemd-udev`?' 

'oh, I replaced `pkg: bash` but forgot of `pkg: cronie` in `template/osfingermap.yaml`. Is it a requirement or an example?'